ansys/dpf/core/server.py

```diff
@@ -685,11 +686,22 @@ def check_ansys_grpc_dpf_version(server, timeout=10.):
     server_version = server.version
     right_grpc_module_version = server_to_ansys_grpc_dpf_version.get(server_version, None)
     if right_grpc_module_version and right_grpc_module_version != grpc_module_version:
-        raise ImportWarning(f"2022R1 Ansys unified install is available. "
-                            f"To use DPF server from Ansys "
-                            f"{server_to_ansys_version.get(server_version, 'Unknown')}"
-                            f" (dpf.SERVER.version=='{server_version}'), "
-                            f"install version {right_grpc_module_version} of ansys-grpc-dpf"
+        compatibility_link = r"https://dpfdocs.pyansys.com/getting_started/index.html#client-server-compatibility"
+        raise ImportWarning(f"An incompatibility has been detected between the server version"
+                            f"({server_version} " 
+                            f"from Ansys {server_to_ansys_version.get(server_version, 'Unknown')})"
+                            f" and the ansys-grpc-dpf version installed ({grpc_module_version})"
+                            f"To follow the compatibility guidelines given in "
+                            f"{compatibility_link} and use DPF server {server_version}, "
+                            f"please install version {right_grpc_module_version} of ansys-grpc-dpf"
                             f" with the command: \n"
                             f"     pip install ansys-grpc-dpf=={right_grpc_module_version}"
                             )
+        # raise ImportWarning(f"2022R1 Ansys unified install is available. "
```

Comment:
@PProfizi the goal of this message is also to inform the user that a newer Ansys version is available and can be installed. This is important to mention

Reply:
@rlagha , ok, noted, I modified the message to:

An incompatibility has been detected between the DPF server version (2.0 from Ansys 2021R2) and the ansys-grpc-dpf version installed (0.4.0). Please consider using the newest DPF server available in the 2022R1 Ansys unified install.
To follow the compatibility guidelines given in https://dpfdocs.pyansys.com/getting_started/index.html#client-server-compatibility while still using DPF server 2.0, please install version 0.3.0 of ansys-grpc-dpf with the command: 
     pip install ansys-grpc-dpf==0.3.0